pages/bars/index.js

```diff
@@ -0,0 +1,21 @@
+import AppHeader from "../../components/AppHeader/index.js";
```

Comment:
Usually, you don't need to put the filename in the import statement if your component is defined in `index.js`, so e.g. 
```js
import AppHeader from "../components/AppHeader"
```
should suffice.